de.dashup.application/src/main/webapp/components/message-broker.js

```diff
@@ -0,0 +1,13 @@
+class MessageBroker {
```

Comment:
What is this class for?

Reply:
This is just a helper class to export logic that does not belong to characteristics of a component to.